addons/mail/static/src/components/attachment/attachment.xml

```diff
@@ -7,7 +7,7 @@
                 'o-downloadable': props.isDownloadable,
                 'o-editable': props.isEditable,
                 'o-has-card-details': detailsMode === 'card',
-                'o-temporary': attachment and attachment.isTemporary,
+                'o-uploading': attachment and attachment.isUploading,
```

Comment:
`o-isUploading` (it matches with naming convention I'm introducing with my WIP PR)

Reply:
Done